src/main.cpp

```diff
@@ -10,6 +11,11 @@ using json = nlohmann::json;
 
 int main(int argc, char const *argv[])
 {
+    if(!fs::exists("../config.json")) {
+        std::cout << canonical(fs::path("..")).append("config.json").string() << " does not exist!" << std::endl;
```

Comment:
Did you try to compile this? `canonical` is under `std::filesystem` yet, it's being used directly? Also, may I ask for your reading behind not just `std::cout << "../config.json doesn't exist\n";` and opting to use std::filesystem here?

Reply:
Yes I did try to compile this. It builds and runs fine for me (see attached screenshot).
My reasoning for using canonical is to print the absolute file path, to reduce user confusion ("../config.json, but where is ..??").

![image](https://user-images.githubusercontent.com/13570458/210164497-bd41b7c3-9609-4cf0-98b8-045123b857ef.png)